src/main/java/com/alipay/oceanbase/rpc/stream/ObTableClientQueryAsyncStreamResult.java

```diff
@@ -181,6 +181,29 @@ protected Map<Long, ObPair<Long, ObTableParam>> refreshPartition(ObTableQuery ta
         return buildPartitions(client, tableQuery, tableName);
     }
 
+    public void renewLease() throws Exception {
```

Comment:
maybe we can add some comments or annotations to explain this function is private and only for hbase query ?